developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere.md

```diff
@@ -48,15 +50,20 @@ services:
 ...
 ```
 
-* Note: you can also use the [Weaviate configuration tool](../../installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+:::note
+* You can also use the [Weaviate configuration tool](../../installation/docker-compose.md#configurator) to create a Weaviate setup with this module.
+* Starting with `v1.11.0` the `COHERE_APIKEY` variable is now optional and you can instead provide the key at insert/query time as an HTTP header.
```

Comment:
* Should we mention the name of the HTTP header?
* Should we consistently format version numbers as code? The vast majority I've seen were plain text.

Reply:
1) Good point - I've edited this bit. 
2) Personally I prefer it as code.